### DISCUSSION TITLE
Targetted speech extraction

### DISCUSSION THREAD

**Discussion by StuartIanNaylor** [pyannote/pyannote-audio#1225] [Ideas]
Just wondered (fingers crossed) if you had any plans for some form of targetted speech extraction in a similar vain to voice-filter-lite?

Thanks

**Comment by hbredin** [comment]
Nope. But such a contribution would be more than welcome :)

**Reply by StuartIanNaylor** [reply]
Only opensource I know is https://github.com/BUTSpeechFIT/speakerbeam but apols as beyond me as why the fingers where crossed :)
